packages/@uppy/react/src/useUppyState.js

```diff
@@ -0,0 +1,19 @@
+import { useSyncExternalStore, useCallback } from 'react'
+
+export default function useUppyState (uppy, selector) {
+  const subscribe = useCallback(
+    () => uppy.store.subscribe(),
```

Comment:
don't we need arguments here? or use `uppy.store.subscribe.bind` as in the original commit?